frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -234,9 +234,15 @@ export const billingLogic = kea<billingLogicType>([
                         message: `You have exceeded the usage limit for ${productOverLimit.name}. Please 
                             ${productOverLimit.subscribed ? 'increase your billing limit' : 'upgrade your plan'}
                             or data loss may occur.`,
+                        dismissKey: 'usage-limit-exceeded',
```

Comment:
```suggestion
                        dismissKey: `usage-limit-exceeded-${billing.billing_period.current_period_end}`,
```

When their billing period turns over, it'll have a different date, so will show up again. 

I did not test this, so try locally before merging ;)

Reply:
Is there a loophole here where a user dismisses this alert, increases their billing limit, and exceeds their limit within the same billing period? They wouldn't see this alert the second time.

Reply:
The logic for resetting this dismissKey every time we don't show this usage exceeded alert would catch that case.